main/src/ca/uwaterloo/flix/language/phase/Lowering.scala

```diff
@@ -698,7 +698,7 @@ object Lowering extends Phase[Root, Root] {
       case _: Type.Ascribe => throw InternalCompilerException(s"Unexpected type: '$tpe0'.")
     }
 
-    if (tpe0.kind == Kind.Schema)
+    if (tpe0.typeConstructor.contains(TypeConstructor.Schema))
```

Comment:
Type constructor is `None` if the base type is a variable. I think that is still possible here, since in some circumstances Monomorph still isn't getting rid of all the type variables (notably in enums).

Reply:
Okay. hmm. We will roll with it.